src/ode/owl_ode.ml

```diff
@@ -3,3 +3,4 @@ module Ode = Owl_ode_base.Ode
 module Types = Owl_ode_base.Types
 module Native = Native
 module Symplectic = Symplectic
+include Types
```

Comment:
How much will this mess up the types? Types from different modules would become different right? I'd rather keep it spearate as before

Reply:
yea that's true I didn't think of that. I'll revert it

Reply:
If you want, you could add `owl_ode.mli` and expose the types there by hand. In the implementation you would just have `val tspec = Types.tspec` while in the signature you can fully expand it (you will also have to copy-paste the docstring but I guess we can live with some level of duplication). I think that should work.

Reply:
argh of course...that would be the way to do it. But perhaps, this is best left for another PR.

Reply:
I don’t know. It’s such a small change that we may as well add it here. Do you think it would take too long?

Reply:
wouldn't we have to change the docs as well??

Reply:
but yea to be fair, we'll probably have to update the docs anyway. okay, I'll make the changes expose the type `tspec`

Reply:
I have added a commit that solves this issue. I have also rebased on the current origin to cleanup the history from the many merges